java/test/jmri/implementation/LightControlTest.java

```diff
@@ -21,22 +21,22 @@
 
     @Test
     public void testCtor() {
-        LightControl l = new LightControl();
-        Assert.assertNotNull("LightControl not null", l);
+        lc = new LightControl();
```

Comment:
Suggestion: add the creation of this light control to the setUp method.  You can then remove it from the individual tests.